src/Themes/HackathonTheme/Views/Widget-TeamDashboardWidget.liquid

```diff
@@ -0,0 +1,98 @@
+{% if User | is_in_role: "Hacker" %}
```

Comment:
I think these files can stay in the module only

Reply:
I forgot to remove them after the last merge, the good ones are in the module. I just pushed an update.